src/calendar/date/eventeditor/CalendarEventModel.ts

```diff
@@ -538,19 +546,55 @@ export function assembleEditResultAndAssignFromExisting(existingEvent: CalendarE
  * @param identity sequence (default "0") and recurrenceId (default null) are optional, but the uid must be specified.
  */
 export function assignEventIdentity(values: CalendarEventValues, identity: Require<"uid", Partial<CalendarEventIdentity>>): CalendarEvent {
-	return createCalendarEvent({
-		...values,
-		sequence: "0",
-		...identity,
-	})
+	return Object.assign(
+		createCalendarEvent({
+			sequence: "0",
+			organizer: null,
+			attendees: [],
+			description: "",
+			repeatRule: null,
+			location: "",
+			summary: "",
+			endTime: new Date(),
+			uid: null,
+			startTime: new Date(),
+			recurrenceId: null,
+			invitedConfidentially: null,
+			// @ts-ignore
+			alarmInfos: null, // FIXME
+			hashedUid: null,
+		}),
+		values,
+		identity,
+	)
 }
 
 async function resolveAlarmsForEvent(alarms: CalendarEvent["alarmInfos"], calendarModel: CalendarModel, user: User): Promise<Array<AlarmInterval>> {
 	const alarmInfos = await calendarModel.loadAlarms(alarms, user)
 	return alarmInfos.map(({ alarmInfo }) => parseAlarmInterval(alarmInfo.trigger))
 }
 
-function cleanupInitialValuesForEditing(initialValues: Partial<CalendarEvent>): CalendarEvent {
+function makeEmptyCalendarEvent(): StrippedEntity<CalendarEvent> {
+	return {
+		// @ts-ignore
+		alarmInfos: null, // FIXME
```

Comment:
fixme?

Reply:
Yeah, this is keeping previous behavior, but we are not sure if it was intended. I think calendar experts should know.